packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -3049,6 +3050,7 @@ abstract class Element extends DiagnosticableTree implements BuildContext {
       _widget = widget;
 
   Element? _parent;
+  DebugReassembleConfig? _debugReassembleConfig;
```

Comment:
Does adding this additional field to `Element` have any effect on release-mode memory footprint? Asked a different way, will this field be tree-shaken in a release build?

Reply:
This field is tree shaken, I inspected the size analysis for hello_world and it was not retained